build fails due to libssh dependency

The build fails with the following error message:
```
make[2]: *** No rule to make target `libssh-prefix/lib/libssh.so', needed by `migfra_benchmark'.  Stop.
```

I could fix this calling: 
```
make -f CMakeFiles/Makefile2 libssh
```

in the `build` folder.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,7 +66,6 @@ ExternalProject_Add(
 ExternalProject_Get_Property(libssh install_dir)
 include_directories(SYSTEM "${install_dir}/include")
 list(APPEND LIBS "${install_dir}/lib/libssh.so")
-list(APPEND LIBS_BENCHMARK "${install_dir}/lib/libssh.so")
 #add_definitions(-DLIBSSH_STATIC=1)
 
 
@@ -91,12 +90,7 @@ set(SRC ${PROJECT_SOURCE_DIR}/src/main.cpp
 	${PROJECT_SOURCE_DIR}/src/pci_device_handler.cpp
 	${PROJECT_SOURCE_DIR}/src/utility.cpp
 )
-set(SRC_BENCHMARK ${PROJECT_SOURCE_DIR}/src/benchmark_main.cpp
-	${PROJECT_SOURCE_DIR}/src/task.cpp
-	${PROJECT_SOURCE_DIR}/src/pscom_handler.cpp
-	${PROJECT_SOURCE_DIR}/src/pci_device_handler.cpp
-	${PROJECT_SOURCE_DIR}/src/utility.cpp
-)
+set(SRC_BENCHMARK ${PROJECT_SOURCE_DIR}/src/benchmark_main.cpp)
 
 ### Add config files
 # Doxygen documentation generation

